diff --git a/include/experimental/__p1673_bits/blas2_matrix_vector_product.hpp b/include/experimental/__p1673_bits/blas2_matrix_vector_product.hpp
index 20071ec..958587d 100644
--- a/include/experimental/__p1673_bits/blas2_matrix_vector_product.hpp
+++ b/include/experimental/__p1673_bits/blas2_matrix_vector_product.hpp
@@ -274,9 +274,9 @@ void matrix_vector_product(
       std::common_type_t<SizeType_A, SizeType_x>,
       SizeType_y>>;
   for (size_type i = 0; i < A.extent(0); ++i) {
-    y(i) = ElementType_y{};
+    y[i] = ElementType_y{};
     for (size_type j = 0; j < A.extent(1); ++j) {
-      y(i) += A(i,j) * x(j);
+      y[i] += A[i,j] * x[j];
     }
   }
 }
diff --git a/include/experimental/__p1673_bits/blas3_matrix_product.hpp b/include/experimental/__p1673_bits/blas3_matrix_product.hpp
index e47796d..296ffe1 100644
--- a/include/experimental/__p1673_bits/blas3_matrix_product.hpp
+++ b/include/experimental/__p1673_bits/blas3_matrix_product.hpp
@@ -743,9 +743,9 @@ void matrix_product(
 
     for (size_type i = 0; i < C.extent(0); ++i) {
       for (size_type j = 0; j < C.extent(1); ++j) {
-        C(i,j) = ElementType_C{};
+        C[i,j] = ElementType_C{};
         for (size_type k = 0; k < A.extent(1); ++k) {
-          C(i,j) += A(i,k) * B(k,j);
+          C[i,j] += A[i,k] * B[k,j];
         }
       }
     }
diff --git a/include/experimental/__p1673_bits/macros.hpp b/include/experimental/__p1673_bits/macros.hpp
index 997bcf3..66c569f 100644
--- a/include/experimental/__p1673_bits/macros.hpp
+++ b/include/experimental/__p1673_bits/macros.hpp
@@ -18,7 +18,7 @@
 #ifndef LINALG_INCLUDE_EXPERIMENTAL___P1673_BITS_MACROS_HPP_
 #define LINALG_INCLUDE_EXPERIMENTAL___P1673_BITS_MACROS_HPP_
 
-#include "__p1673_bits/linalg_config.h"
+// #include "__p1673_bits/linalg_config.h"
 
 // Work around a known MSVC issue, that by default
 // it always defines __cplusplus as for C++98,
diff --git a/include/experimental/linalg b/include/experimental/linalg
index 009d60f..6fee21f 100644
--- a/include/experimental/linalg
+++ b/include/experimental/linalg
@@ -17,7 +17,7 @@
 
 #pragma once
 
-#include "__p1673_bits/linalg_config.h"
+// #include "__p1673_bits/linalg_config.h"
 #include "__p1673_bits/macros.hpp"
 #include "__p1673_bits/linalg_execpolicy_mapper.hpp"
 #include "__p1673_bits/maybe_static_size.hpp"
